build_wheel.sh

```diff
@@ -0,0 +1,176 @@
+  __test_build
+  __teardown
+}
+
+__validate_files() {
+  echo "Validating files ... "
+  # venv
+  if [[ ! -e "$(command -v python3)" ]]; then
+    echo "Is python installed?" 1>&2
+    return 1
+  fi
+
+  # setup.py
+  if [[ ! -e "${SETUP_FILE}" ]]; then
+    echo "${SETUP_FILE} file not found" 1>&2
+    return 3
+  fi
+}
+
+__copy_files() {
+  echo "Copying files ... "
+
+  # create empty yolact dir, overwrite if exists
+  [[ -d ${REPO_NAME} ]] && rm --recursive ${REPO_NAME}
+  mkdir --parents "${REPO_NAME}"
+
+  for folder in "${NEEDED_FOLDERS[@]}"; do
+    cp --recursive "${folder}" "${REPO_NAME}"
+  done
+
+  echo "Copying ${REQUIREMENTS_FILE} ... "
+  cp "${REQUIREMENTS_FILE}" "${REPO_NAME}"
+
+  echo "Finding and Copying Python Files ... "
+  find . -maxdepth 1 -type f -name "*${PYTHON_EXTENSION}" -exec cp {} "${REPO_NAME}" \;
+
+}
+
+__fix_requirements() {
+  echo "Fixing ${REQUIREMENTS_FILE} ... "
+  sed -i '/^sparseml/d' "${REPO_NAME}/${REQUIREMENTS_FILE}"
+
+  yolact_file=".*yolact.py"
+  echo "Fixing file level imports ... "
+  find "${REPO_NAME}" -maxdepth 1 -type f -name "*${PYTHON_EXTENSION}" -print0 | while read -r -d $'\0' file; do
+    echo "file is $file"
+    if ! [[ "${file}" =~ $yolact_file ]]; then
+      __fix_imports "${file}"
+    fi
+  done
+
+  echo "Fixing folder level imports ... "
+  for folder in "${NEEDED_FOLDERS[@]}"; do
+    __fix_imports "${folder}"
+  done
+
+  echo "Fixing special case (yolact.py) imports ... "
+  grep --include="*.py" -rnl "${REPO_NAME}/" -e '\(from\|import\) yolact ' | xargs -i@ sed -E -i "s/^(from|import) yolact /\1 ${REPO_NAME}.yolact /g" @
+
+}
+
+__fix_imports() {
+  curr_import_name=$(echo "${1}" | sed -E "s/.*\/(.*)\.py$/\1/")
+
+  echo "Fixing top level imports for ${curr_import_name} ... "
+  grep --include="*.py" -rnl "${REPO_NAME}/" -e "import ${curr_import_name}" | xargs -i@ sed -i "s/^import ${curr_import_name} /import ${REPO_NAME}.${curr_import_name} /g" @
+
+  echo "Fixing from imports for ${curr_import_name} ... "
+  grep --include="*.py" -rnl "${REPO_NAME}/" -e "from ${curr_import_name}" | xargs -i@ sed -E -i "s/^from ${curr_import_name}([\\.| ])/from ${REPO_NAME}.${curr_import_name}\1/g" @
+
+}
+
+__build_setup() {
+  # install build-tools
+  python3 -m pip install --upgrade build
+
+  package_name="$(python3 ${SETUP_FILE} --name)"
+  package_version="$(python3 ${SETUP_FILE} --version)"
+
+  if [[ -z "$package_name" || -z "$package_version" ]]; then
+    echo "Could not determine package name/version (found ${package_name}==${package_version})" 1>&2
+    return 4
+  fi
+  echo "Package: ${package_name}==${package_version}"
+
+  # Cleanup old dist folder
+  if [[ -e dist ]]; then
+    echo "Removing old releases in dist/* ... "
+    rm --recursive --force dist/ || return 5
+  fi
+
+  echo "NM_INTEGRATED=True" >>"${REPO_NAME}/__init__.py"
+}
+
+__test_build() {
+  # shellcheck disable=SC2174
+  mkdir --mode 700 --parents "${PACKAGE_TEST_DIR}" || return 11
```

Comment:
You might consider attempting to delete the dir before using it. You could also consider `mktemp -d`... I've never used it but could be an option https://www.mktemp.org/ (referred from https://stackoverflow.com/questions/4632028/how-to-create-a-temporary-directory)